### PR TITLE
Separate source/pkg builds

### PR DESCRIPTION
**Pull request by YakDriver**

### CODE REVIEW

**Comment by YakDriver** [comment]
go codebuild go

**Comment by YakDriver** [comment]
With this PR, Terrafirm will only create standalone test instances if the appropriate builder succeeds. The source builds begin immediately without concern for the builder or packages.

File: main.tf [hunk 1]
```diff
@@ -141,45 +141,158 @@ resource "aws_instance" "win" {
...
+  user_data                   = <<-HEREDOC
+    <powershell>
+    ${element(data.template_file.win_pkg_script_preface.*.rendered, count.index)}
+    ${join("", data.template_file.win_userdata_common.*.rendered)} ${join("", data.template_file.win_userdata_specific.*.rendered)}
+    </powershell>
+    HEREDOC
+  vpc_security_group_ids      = [join("", aws_security_group.winrm_sg.*.id)]
+
+  tags = {
+    Name = "${local.resource_name}-${element(local.win_pkg_requests, count.index)}-${join("", aws_instance.win_builder.*.id)}(builder id)"
```
[the remark this answers is not in the document]
Comment (YakDriver) [reply main.tf]: I like it

**Comment by lorengordon** [comment]
go codebuild go

**Pull request merged**